certified-connectors/Easyvista_Service._Manager/apiProperties.json

```diff
@@ -0,0 +1,56 @@
+{
+  "properties": {
+    "capabilities": [],
+    "publisher": "EasyVista",
+    "stackOwner": "Cédric MARTIN / Product Team",
```

Comment:
@ezv-compteintegration for all our verified partners we ask to match the "publisher" and "stackOwner".
Please change the stackOwner to "EasyVista"